guides/cerebras.md

```diff
@@ -0,0 +1,239 @@
+# Cerebras AI Suite Provider Guide
+
+## About Cerebras
+
+At Cerebras, we've developed the world's largest and fastest AI processor, the Wafer-Scale Engine-3 (WSE-3). The Cerebras CS-3 system, powered by the WSE-3, represents a new class of AI supercomputer that sets the standard for generative AI training and inference with unparalleled performance and scalability.
+
+With Cerebras as your inference provider, you can:
+- Achieve unprecedented speed for AI inference workloads
+- Build commercially with high throughput
+- Effortlessly scale your AI workloads with our seamless clustering technology
+
+Our CS-3 systems can be quickly and easily clustered to create the largest AI supercomputers in the world, making it simple to place and run the largest models. Leading corporations, research institutions, and governments are already using Cerebras solutions to develop proprietary models and train popular open-source models.
+
+Want to experience the power of Cerebras? Check out our [website](https://cerebras.net) for more resources and explore options for accessing our technology through the Cerebras Cloud or on-premise deployments!
+
+> [!NOTE]  
+> This SDK has a mechanism that sends a few requests to `/v1/tcp_warming` upon construction to reduce the TTFT. If this behaviour is not desired, set `warm_tcp_connection=False` in the constructor.
+>
+> If you are repeatedly reconstructing the SDK instance it will lead to poor performance. It is recommended that you construct the SDK once and reuse the instance if possible.
+
+## Documentation
+
+The REST API documentation can be found on [inference-docs.cerebras.ai](https://inference-docs.cerebras.ai).
+
+
+## Usage
+Get an API Key from [cloud.cerebras.ai](https://cloud.cerebras.ai/) and add it to your environment variables:
+
+```shell
+export CEREBRAS_API_KEY="your-cerebras-api-key"
+```
+
+Use the python client.
+
+```python
+import aisuite as ai
+client = ai.Client()
+
+models = "cerebras:llama3.1-8b"
```

Comment:
you defined this as `models` but `models` is never used anywhere in the code after this